lib/site_settings/validations.rb

```diff
@@ -173,6 +173,9 @@ def validate_s3_backup_bucket(new_val)
   end
 
   def validate_enforce_second_factor(new_val)
+    if SiteSetting.enable_sso == true
```

Comment:
```suggestion
    if SiteSetting.enable_sso?
```
Any particular reason why you were checking if `== true`?

Reply:
No particular reason. It was an oversight on my part.